lisp/ess-julia.el

```diff
@@ -258,7 +258,7 @@ objects from that MODULE."
 
 
 ;;; ELDOC
-(defun ess-julia-eldoc-function ()
+(defun ess-julia-eldoc-function (&optional callback)
```

Comment:
Can you make these `&rest _ignored` as in the emacs-source commit you've linked to, please?

Reply:
Yes that makes sense, I've made the change now